modules/persistent-storage-csi-vol-populator-procedure-admin.adoc

```diff
@@ -0,0 +1,192 @@
+metadata:
+  name: hello-account
+  namespace: hello <1>
+----
+<1> Reference the namespace that you created earlier.
+
+.. Create a cluster role for the populator using the following example YAML file:
++
+.Example cluster role YAML file
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hello-role
+rules:
+  - apiGroups: [hello.example.com]
+    resources: [hellos]
+    verbs: [get, list, watch]
+----
+
+.. Create a cluster role binding using the following example YAML file:
++
+.Example cluster role binding YAML file
+[source,yaml]
+----
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hello-binding <1>
+subjects:
+  - kind: ServiceAccount
+    name: hello-account <2>
+    namespace: hello <3>
+roleRef:
+  kind: ClusterRole
+  name: hello-role <4>
+  apiGroup: rbac.authorization.k8s.io
+----
+<1> Role binding name.
+<2> Reference the name of the service account that you created earlier.
+<3> Reference the name of the namespace for the service account that you created earlier.
+<4> Reference the cluster role you created earlier.
+
+.. Create a Deployment for the populator using the following example YAML file:
++
+.Example deployment YAML file
+[source,yaml]
+----
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: hello-populator
+  namespace: hello <1>
+spec:
+  selector:
+    matchLabels:
+      app: hello
+  template:
+    metadata:
+      labels:
+        app: hello
+    spec:
+      serviceAccount: hello-account <2>
+      containers:
+        - name: hello
+          image: registry.k8s.io/sig-storage/hello-populator:v1.0.1
+          imagePullPolicy: IfNotPresent
+          args:
+            - --mode=controller
+            - --image-name=registry.k8s.io/sig-storage/hello-populator:v1.0.1
+            - --http-endpoint=:8080
+          ports:
+            - containerPort: 8080
+              name: http-endpoint
+              protocol: TCP
+----
+<1> Reference the namespace that you created earlier.
+<2> Reference the service account that you created earlier.
+
+. Create a volume populator to register the `kind:Hello` resource as a valid data source for the volume using the following example YAML file:
++
+.Example volume populator YAML file
+[source,yaml]
+----
+kind: VolumePopulator
+apiVersion: populator.storage.k8s.io/v1beta1
+metadata:
+  name: hello-populator <1>
+sourceKind:
+  group: hello.example.com
+  kind: Hello
+----
+<1> Volume populator name.
++
+PVCs that use an unregistered populator generate an event: "The datasource for this PVC does not match any registered VolumePopulator", indicating that the PVC might not be provisioned because you are using an unknown (unregistered) populator. 
+
+.Next steps
+* You can now create CR instances of this CRD to populate PVCs. 
++
+For information about how to pre-populate volumes using a volume populator, see Section _Creating pre-populated volumes with volume populators_.
```

Comment:
Section naming mismatch -> here we say "pre-populated" and the actual section says "prepopulated": `Creating prepopulated volumes using volume populators`